src/wasm-interpreter.h

```diff
@@ -1874,6 +1874,9 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
           trap("not an i31");
         }
         break;
+      case ExternInternalize:
+      case ExternExternalize:
+        WASM_UNREACHABLE("unimplemented extern conversion");
```

Comment:
I think we can just do a no-op here?
```suggestion
        // In a production VM interesting things happen here, but all references
        // are the same for us.
        break;
```

Reply:
We don't currently have a way of representing externref values, and unlike the real `ref.as_*`, this is a conversion rather than a downcast, so I think we'll need to at least change the type of the `Literal` value we return.